packages/common/src/hooks/useChangeEmailFormConfiguration.ts

```diff
@@ -110,16 +125,21 @@ export const useChangeEmailFormConfiguration = (onComplete: () => void) => {
         })
         onComplete()
       } catch (e) {
+        console.warn(e)
         if (isOtpMissingError(e)) {
           helpers.setFieldTouched('otp', false)
           setPage(ChangeEmailPage.VerifyEmail)
-        } else {
+        } else if (isInvalidCredentialsError(e)) {
           helpers.setFieldError('otp', messages.invalidCredentials)
           helpers.setFieldError('email', messages.somethingWrong)
```

Comment:
no log in this case? i guess we're pretty damn sure what the issue is here

Reply:
I don't think we'll glean anything interesting from sentry here - but axiom will have more deetz thanks to the identity side changes!